google/cloud/spanner_dbapi/utils.py

```diff
@@ -71,11 +82,19 @@ def backtick_unicode(sql):
 
 
 def sanitize_literals_for_upload(s):
-    """
-    Convert literals in s, to be fit for consumption by Cloud Spanner.
-    1. Convert %% (escaped percent literals) to %. Percent signs must be escaped when
-    values like %s are used as SQL parameter placeholders but Spanner's query language
-    uses placeholders like @a0 and doesn't expect percent signs to be escaped.
-    2. Quote words containing non-ASCII, with backticks, for example föö to `föö`.
+    """Convert literals in s, to be fit for consumption by Cloud Spanner.
+
+    * Convert %% (escaped percent literals) to %. Percent signs must be escaped
+      when values like %s are used as SQL parameter placeholders but Spanner's query
+      language uses placeholders like @a0 and doesn't expect percent signs to be
+      escaped.
+
+    * Quote words containing non-ASCII, with backticks, for example föö to `föö`.
```

Comment:
Single quotes should wrap both "foo" words, it looks.

Reply:
Nope, the mean of the function to add quotes for non-ASCII words so its like an example.